scipy/optimize/_differentialevolution.py

```diff
@@ -254,6 +255,13 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
         ``'vectorized'``, and when to use ``'workers'``.
 
         .. versionadded:: 1.9.0
+        
+    strategy_func : fn(candidate, population, rng=None)
+        If a custom strategy function is provided, it will be called instead
+        of the default function in order to mutate and/or recombine solution
+        vectors. It should return a new candidate vector.
+        
+        .. versionadded:: 1.11.3
```

Comment:
If merged this will be in 1.12
```suggestion
        .. versionadded:: 1.12.0
```